pkg/agent/flowexporter/exporter/exporter_test.go

```diff
@@ -217,44 +240,51 @@ func getElemList(ianaIE []string, antreaIE []string) []ipfixentities.InfoElement
 	// Need only element name and other fields are set to dummy values
 	elemList := make([]ipfixentities.InfoElementWithValue, len(ianaIE)+len(IANAReverseInfoElements)+len(antreaIE))
 	for i, ie := range ianaIE {
-		elemList[i] = ipfixentities.NewInfoElementWithValue(ipfixentities.NewInfoElement(ie, 0, 0, 0, 0), nil)
+		element, _ := registry.GetInfoElement(ie, ipfixregistry.IANAEnterpriseID)
+		ieWithValue, _ := ipfixentities.DecodeAndCreateInfoElementWithValue(element, nil)
+		elemList[i] = ieWithValue
 	}
 	for i, ie := range IANAReverseInfoElements {
-		elemList[i+len(ianaIE)] = ipfixentities.NewInfoElementWithValue(ipfixentities.NewInfoElement(ie, 0, 0, ipfixregistry.IANAReversedEnterpriseID, 0), nil)
+		element, _ := registry.GetInfoElement(ie, ipfixregistry.IANAReversedEnterpriseID)
+		ieWithValue, _ := ipfixentities.DecodeAndCreateInfoElementWithValue(element, nil)
+		elemList[i+len(ianaIE)] = ieWithValue
 	}
 	for i, ie := range antreaIE {
-		elemList[i+len(ianaIE)+len(IANAReverseInfoElements)] = ipfixentities.NewInfoElementWithValue(ipfixentities.NewInfoElement(ie, 0, 0, 0, 0), nil)
+		element, _ := registry.GetInfoElement(ie, ipfixregistry.AntreaEnterpriseID)
+		ieWithValue, _ := ipfixentities.DecodeAndCreateInfoElementWithValue(element, nil)
+		elemList[i+len(ianaIE)+len(IANAReverseInfoElements)] = ieWithValue
 	}
 
 	for i, ie := range elemList {
-		switch ieName := ie.Element.Name; ieName {
+		switch ieName := ie.GetInfoElement().Name; ieName {
```

Comment:
maybe it would have been nice to define `Name()` and `DataType()` methods on `InfoElementWithValue`. It would improve readability IMO.